src/Illuminate/Database/Query/Builder.php

```diff
@@ -2562,7 +2563,7 @@ public function aggregate($function, $columns = ['*'])
         $results = $this->cloneWithout($this->unions ? [] : ['columns'])
                         ->cloneWithoutBindings($this->unions ? [] : ['select'])
                         ->setAggregate($function, $columns)
-                        ->get($columns);
```

Comment:
Why is this removed? I'm sure this has come up before? // cc @themsaid

Reply:
I removed it because the get method cannot handle subqueries as columns.
I'm not sure why it's needed though, this argument exists since the first commit ever, but the implementation of the `aggregate` method has changed a lot (mostly cloning issues).

I don't believe it's needed, since `runPaginationCountQuery` executes a similar query without this argument on the `get` method.

Reply:
I feel like I can basically guarantee if it is present in the code base it is needed.